tests/dags/providers/provider_api_scripts/test_smk.py

```diff
@@ -14,6 +12,8 @@
     format="%(asctime)s - %(name)s - %(levelname)s:  %(message)s", level=logging.DEBUG
 )
```

Comment:
This logging line can be removed!

Reply:
Nice! I thought about deleting it but seeing that it was in other test files as well (like stocksnap & cleveland) I left it intact. I'll remove it 🧹